example/fuse_wallet_sdk_example.dart

```diff
@@ -1,87 +1,18 @@
-import 'dart:convert';
 import 'dart:io';
```

Comment:
example -> examples?
unless it's a standard in Dart

Reply:
Can you also link in the README.md to the examples folder. So it won't be missed, much of good stuff is there